geniza/footnotes/models.py

```diff
@@ -606,7 +606,9 @@ def content_html(self):
         # keyed on canvas uri
         # handle multiple annotations on the same canvas
         html_content = defaultdict(list)
-        for a in self.annotation_set.all():
+        for a in self.annotation_set.all().order_by(
```

Comment:
probably worth a one-line comment here documenting what this is for

I checked, default ordering is by `created`; makes sense we wouldn't want to rely on an optional/custom parameter for sort